cluster/addons.go

```diff
@@ -0,0 +1,42 @@
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/rancher/rke/k8s"
+	"github.com/rancher/rke/pki"
+)
+
+const (
+	ClusterDnsServerIPEnvName = "RKE_DNS_SERVER"
+	ClusterDomainEnvName      = "RKE_CLUSTER_DOMAIN"
+)
+
+func (c *Cluster) DeployK8sAddOns() error {
+	if err := c.deployKubeDNS(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *Cluster) deployKubeDNS() error {
+	logrus.Infof("[Plugins] Setting up KubeDNS")
```

Comment:
can we change name Plugins to addons and to lowercase to be consistent with other log format in the code